peewee_migrate/router.py

```diff
@@ -153,9 +153,8 @@ def run_one(self, name, migrator, fake=True, downgrade=False, force=False):
 
         except Exception as exc:
             self.database.rollback()
-            self.logger.exception(exc)
```

Comment:
`logger.exception()` does not require to pass exception object if it called in `except` statement, see https://docs.python.org/2/library/logging.html#logging.Logger.exception